From 095792d1a2d0ea58858cdbcd33a5f1923474c04f Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Mon, 29 Feb 2016 01:33:41 +0100
Subject: [PATCH 3/3] d3d11 renderer: fix versionhelpers.h inclusion

Can't even expect people to write file names correctly these days.
---
 src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp b/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp
index 9b1ffa5..1111669 100644
--- a/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp
+++ b/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp
@@ -10,7 +10,7 @@
 
 #include <EGL/eglext.h>
 #include <sstream>
-#include <VersionHelpers.h>
+#include <versionhelpers.h>
 
 #include "common/tls.h"
 #include "common/utilities.h"
-- 
2.7.0

